r/R/util.R

```diff
@@ -215,3 +215,138 @@ handle_csv_read_error <- function(e, schema, call) {
 is_compressed <- function(compression) {
   !identical(compression, "uncompressed")
 }
+
```

Comment:
If these functions aren't used anywhere other than dplyr-funcs-datetime.R, you could just keep them in the same file. util.R is usually for more general utilities.

Reply:
In order to avoid too many of the date/time helpers in dplyr-funcs-datetime.R, I created dplyr-datetime-helpers.R - that's another location where they could live (they don't have to).